### PR TITLE
e2eping support using env variables

### PR DESCRIPTION
**Pull request by prayagsingh**
1. enable/disable e2eping using env variable
2. enable/disable remote video menu using env variable
3. allow disabling private-chat option

### CODE REVIEW

**Comment by saghul** [comment]
Oh I had forgotten about this one. Thanks for the reminder!

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -42,7 +42,8 @@
 {{ $VIDEOQUALITY_ENFORCE_PREFERRED_CODEC := .Env.VIDEOQUALITY_ENFORCE_PREFERRED_CODEC | default "false" | toBool -}}
 {{ $DISABLE_POLLS := .Env.DISABLE_POLLS | default "false" | toBool -}}
 {{ $DISABLE_REACTIONS := .Env.DISABLE_REACTIONS | default "false" | toBool -}}
-
+{{ $DISABLE_REMOTE_VIDEO_MENU := .Env.DISABLE_REMOTE_VIDEO_MENU | default "true" | toBool -}}
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:45]: The default here needs to be false.

File: web/rootfs/defaults/settings-config.js [hunk 2]
```diff
@@ -377,9 +378,26 @@ config.hiddenPremeetingButtons = [ '{{ join "','" (splitList "," .Env.HIDE_PREME
...
+config.remoteVideoMenu.disabled = {{ $DISABLE_REMOTE_VIDEO_MENU }};
 {{ if .Env.DISABLE_KICKOUT -}}
 config.remoteVideoMenu.disableKick = {{ .Env.DISABLE_KICKOUT }};
 {{ end -}}
 {{ if .Env.DISABLE_GRANT_MODERATOR -}}
 config.remoteVideoMenu.disableGrantModerator = {{ .Env.DISABLE_GRANT_MODERATOR }};
-{{ end -}}
\ No newline at end of file
+{{ end -}}
+{{ if .Env.DISABLE_PRIVATE_CHAT -}}
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:388]: While you're here, can you please set these options unconditionally, with defaults?

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

File: web/rootfs/defaults/settings-config.js (hunk 2 above)
Comment (prayagsingh) [reply web/rootfs/defaults/settings-config.js]: Added a defaults like `{{ $DISABLE_PRIVATE_CHAT:= .Env.DISABLE_PRIVATE_CHAT | default "false" | toBool -}}`
And now using `config.remoteVideoMenu.disablePrivateChat = {{ $DISABLE_PRIVATE_CHAT }};`

File: web/rootfs/defaults/settings-config.js [hunk 3]
```diff
@@ -386,9 +388,24 @@ config.hiddenPremeetingButtons = [ '{{ join "','" (splitList "," .Env.HIDE_PREME
 
 // Configure remote participant video menu
 if (!config.hasOwnProperty('remoteVideoMenu')) config.remoteVideoMenu = {};
+config.remoteVideoMenu.disabled = {{ $DISABLE_REMOTE_VIDEO_MENU }};
 {{ if .Env.DISABLE_KICKOUT -}}
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:392]: Can you please apply the same pattern to these other 2 related options?

Comment (prayagsingh) [reply web/rootfs/defaults/settings-config.js:392]: Please check if the defaults are fine
DISABLE_KICKOUT=false
DISABLE_GRANT_MODERATOR=false

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Thank you! Sorry again for the delay.

**Pull request merged**